mitodl/ol-django
# Add necessary migration for geoip app and some minor build tweaks

### What are the relevant tickets?

n/a

### Description (What does it do?)

- At some point some indexes got renamed (from `maxmind` to `geoip`) - this adds a migration for that
- Updates the docker-compose.yml to expose the Postgres DB on a different, configurable port (55432)
- Update the README.md to add Postgres update and testapp command running detail

### How can this be tested?

Automated tests should pass. This just changes the names of some indexes so there's no functional change to test.

For the changes to the build setup,
- The documentation in README.md should make sense.
- Running the container should not cause a port conflict if you're running something else that has Postgres on port 5432.
- You should be able to adjust the port for the Postgres container.
